get_aws_secrets.sh

```diff
@@ -1,35 +1,41 @@
 #!/bin/bash
 
-[ -z `which aws-okta` ] && echo "Please run 'brew install aws-okta' in order to operate this script" && exit 1
 [ -z `which jq` ] && echo "Please run 'brew install jq' in order to operate this script" && exit 1
-[ -z "$1" ] && echo -e "Error! aws_account_profile is missing!\n\n  ex: $0 <aws_account_profile>\n" && exit 1 
 
-aws_account_profile=$1
+if ! [[ $(env | grep "AWS_") ]]; then
```

Comment:
This check will not work if AWS_PROFILE is set

```
bash-3.2$ export AWS_PROFILE=sso_sandbox_admin
bash-3.2$ if ! [[ $(env | grep "AWS_") ]]; then
>   echo "Please set required AWS environment variables"
>   echo "[AWS SSO] Login to AWS SSO portal and copy/paste environment variables"
>   echo "[DEPRECATED aws-okta] Run 'aws-okta exec <aws_account_profile> -- \$SHELL'"
>   exit 1
> fi
bash-3.2$ echo $?
0
bash-3.2$ unset AWS_PROFILE
bash-3.2$ if ! [[ $(env | grep "AWS_") ]]; then   echo "Please set required AWS environment variables";   echo "[AWS SSO] Login to AWS SSO portal and copy/paste environment variables";   echo "[DEPRECATED aws-okta] Run 'aws-okta exec <aws_account_profile> -- \$SHELL'";   exit 1; fi
Please set required AWS environment variables
[AWS SSO] Login to AWS SSO portal and copy/paste environment variables
[DEPRECATED aws-okta] Run 'aws-okta exec <aws_account_profile> -- $SHELL'
exit
```

Reply:
Added `| grep -E "ACCESS_KEY|SESSION_TOKEN")` to the command